src/api/db/data/20200420080753_generate_web_notifications.rb

```diff
@@ -0,0 +1,35 @@
+class GenerateWebNotifications < ActiveRecord::Migration[6.0]
+  def up
+    Notification.update(rss: true, web: true)
+    generate_event_subscripitions
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  private
+
+  def generate_event_subscripitions
+    event_subscriptions = EventSubscription.where(channel: :instant_email).where.not(user_id: nil)
+    event_subscriptions.each do |event_subscription|
+      create_subscription_for_channel(event_subscription, :rss)
```

Comment:
Hm. I think we only should do this if the subscription user has a `Token::Rss`

Reply:
Done, we don't create subscriptions if the user doesn't have rss token